coretex/utils/number.py

```diff
@@ -39,3 +39,25 @@ def mathematicalRound(value: float, decimalPlaces: int) -> float:
     places = Decimal(10) ** -decimalPlaces
 
     return float(decimal.quantize(places, rounding = ROUND_HALF_UP))
+
+
+def formatBytes(value: int, precision = 2):
```

Comment:
Explicitly annotate type of `precision` and return type of the function.

Reply:
Added.